tests/scripts/thread-cert/node.py

```diff
@@ -2592,6 +2592,55 @@ class LinuxHost():
     PING_RESPONSE_PATTERN = re.compile(r'\d+ bytes from .*:.*')
     ETH_DEV = config.BACKBONE_IFNAME
 
+    def enable_ether(self):
+        """Enable the ethernet interface.
+        """
+
+        self.bash(f'ifconfig {self.ETH_DEV} up')
+
+    def disable_ether(self):
+        """Disable the ethernet interface.
+        """
+
+        self.bash(f'ifconfig {self.ETH_DEV} down')
+
+    def add_ether_address(self, scope, addr):
+        """Add IPv6 address to the ethernet interface.
+
+        :param scope: the scope of the address, e.g. 'scope', 'global'.
+        : param addr: the IPv6 address including the trailing prefix
+                      length separated with '/'.
+
+        Example usage of adding a link-local address:
+            self.add_ether_address('link', 'fe80::42:c0ff:fea8:903/64')
+        """
+
+        self.bash(f'ip addr add dev {self.ETH_DEV} scope {scope} {addr}')
+
+    def remove_ether_address(self, scope, addr):
```

Comment:
If this method is useless, remove it?

Reply:
Ethernet address related method are all removed now.